Packs/TOPdesk/Integrations/TOPdesk/README.md

```diff
@@ -51,7 +51,32 @@ Make sure you use the right account for your needs and that the account used has
 #### Troubleshooting
 Make sure the application password is not expired by logging in TOPdesk and viewing it as described in step 3 of the **Setup TOPdesk's application password** procedure. 
 
-### FIQL query
+### Incident Mirroring
+
+You can enable incident mirroring between Cortex XSOAR incidents and TOPdesk incidents.
+
+To setup the mirroring follow these instructions:
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for **TOPdesk** and select your integration instance.
+3. Enable **Fetches incidents**.
+4. Optional: You can go to the *Incidents fetch query* or *Detections fetch query* parameter and select the query to fetch the incidents from TOPdesk.
+5. In the *Mirroring Direction* integration parameter, select in which direction the incidents should be mirrored:
+    - Incoming - Any changes in TOPdesk incidents (`state`, `status`, `tactics`, `techniques`, `objectives`, `tags`, `hosts.hostname`) will be reflected in XSOAR incidents.
+    - Outgoing - Any changes in XSOAR incidents will be reflected in TOPdesk incidents (`tags`, `status`).
+    - Incoming And Outgoing - Changes in XSOAR incidents and TOPdesk incidents will be reflected in both directions.
+    - None - Turns off incident mirroring.
+6. In the *Comment Entry Tag* integration parameter, you can set the tag which can be added to comments to mirror from XSOAR to TOPdesk. The comment in TOPdesk will be visible for the person and the operator. 
+7. In the *Work Note Entry Tag* integration parameter, you can set the tag which can be added to comments to mirror from XSOAR to TOPdesk. The work note in TOPdesk will only be visible for the operator and not the person. 
+8. In the *File Entry Tag* integration parameter, you can set the tag which can be added to a file to mirror the file from XSOAR to TOPdesk. The file in TOPdesk will only be visible for the operator and not the person. 
```

Comment:
I will be happy to see this in the demo. 6 to 8 can be merged into one instruction.

Reply:
merged into 1 instruction

Reply:
done